packages/client/src/App.sass

```diff
@@ -412,6 +412,9 @@ input:focus-visible
   flex: 2 0 0
 .flex-3
   flex: 3 0 0
+.flex-9
+  flex: 9 0 0
```

Comment:
Should probably set to flex: 9 9 9 (or flex:9 in short) if we want this column take 9 spaces on all screen sizes